webpack.dev.js

```diff
@@ -7,5 +7,6 @@ module.exports = merge(common, {
     devServer: {
         port: 1234,
         hot: true,
+        historyApiFallback: true,
```

Comment:
Updating our _**webpack config**_ to allow us to handle invalid paths ([docs here](https://webpack.js.org/configuration/dev-server/#devserverhistoryapifallback))

Reply:
Will review the webpack wiki but see then addition of the historyApiFallback set to true, what is the **hot** option? 🔥 🔥 🔥 ???

Reply:
Hot option is for [Hot module replacement](https://webpack.js.org/concepts/hot-module-replacement/) - reload the parts of the application that have changed (i.e. when you modify the code, and have the local dev server running, you will notice that webpack notices the changes and starts to rebuild automatically)